presto-native-execution/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${KNOWN_WARNINGS}")
 # Add all Presto options below
 option(PRESTO_ENABLE_S3 "Build S3 connector" OFF)
 option(PRESTO_ENABLE_HDFS "Build HDFS connector" OFF)
-option(PRESTO_ENABLE_PARQUET "Enable Parquet support" OFF)
+option(PRESTO_ENABLE_PARQUET "Enable Parquet support" ON)
```

Comment:
Is a change needed at this level ? Ivan had previously checked in https://github.com/prestodb/presto/pull/19476. Is that sufficient to just build test-native with Parquet. Does that not work for this PR ?

Reply:
@aditi-pandit Nice catch. This is not needed after #19476 lands.